code/modules/overmap/spacetravel.dm

```diff
@@ -5,31 +5,35 @@
 	known = 0
 	in_space = TRUE
 
-/obj/effect/overmap/visitable/sector/temporary/New(var/nx, var/ny)
+/obj/effect/overmap/visitable/sector/temporary/Initialize(var/nx, var/ny)
 	loc = locate(nx, ny, global.using_map.overmap_z)
 	x = nx
 	y = ny
 	var/emptyz = global.using_map.get_empty_zlevel()
 	map_z += emptyz
 	map_sectors["[emptyz]"] = src
 	testing("Temporary sector at [x],[y] was created, corresponding zlevel is [emptyz].")
+	return ..()
```

Comment:
You're going to want to do `. = ..()` on line one so that the base Initialize() proc executes and populates `map_z`.